frontend/packages/ceph-storage-plugin/console-extensions.json

```diff
@@ -198,5 +198,12 @@
         "$codeRef": "odfSystemDashboard.default"
       }
     }
+  },
+  {
+    "type": "console.action/provider",
+    "properties": {
+      "contextId": "csv-actions",
```

Comment:
Any reason to use a new context id for CSVs? Wouldn't CSV resource actions work here?

Reply:
Why not use `console.action/resource-provider` extension?

Reply:
we have common actions `Delete, Edit yaml` that shows up on each operator backed resources that's why used a different context menu https://github.com/openshift/console/pull/9998/files#diff-9993fe676fb030dc047b550a63b46524797154b76f9ff27cf33d8abc24233081R11-R12